integration_tests/expected_data/sars_cov_2/illumina_unknown/results.csv

```diff
@@ -1,7 +1,7 @@
-5695c67c-b7ac-4fc2-96f0-f1ffc68c03fb,Completed,9038,Midnight-ONT_V3,8439,29,0,1.0,unknown,0.79,99.17,22147,459268,True,BA.1,0.0,,Omicron (BA.1-like),0.92,0.0,scorpio call: Alt alleles 53; Ref alleles 0; Amb alleles 2; Oth alleles 3,PUSHER-v1.26,4.3,0.3.17,v0.1.12,False,pass,Ambiguous_content:0.02,Usher placements: BA.1(1/1)
-70fd6fd3-6eed-48a4-8e5d-d59ecb1dbf32,Completed,1990720,NEB-VarSkip_v2b,4640,1,0,0.0133333333333333,unknown,0.0,0.0,0,0,False,Unassigned,,,,,,,PUSHER-v1.26,4.3,0.3.17,v0.1.12,False,fail,failed to map,
-9729bce7-f0a9-4617-b6e0-6145307741d1,Completed,644,ARTIC_V4-1,210,39,0,0.3786407766990291,unknown,1.44,98.55,21625,265859,True,AY.6,0.0,,Delta (B.1.617.2-like),0.77,0.08,scorpio call: Alt alleles 10; Ref alleles 1; Amb alleles 2; Oth alleles 0,PUSHER-v1.26,4.3,0.3.17,v0.1.12,False,pass,Ambiguous_content:0.03,Usher placements: AY.6(1/1)
-aff01b7c-8a0a-40bf-927a-0548add8ea5a,Completed,696,ARTIC_v5-2-0-400,278,53,0,0.4907407407407407,unknown,2.48,97.45,10310,273604,True,AY.98,0.0,,Delta (B.1.617.2-like),1.0,0.0,scorpio call: Alt alleles 13; Ref alleles 0; Amb alleles 0; Oth alleles 0,PUSHER-v1.26,4.3,0.3.17,v0.1.12,False,pass,Ambiguous_content:0.04,Usher placements: AY.98(1/1)
+SAMPLE_ID,STATUS,CONTAMINATED_READS,PRIMER_DETECTED,PRIMER_NUMREADS,PRIMER_UNIQUE_NUMREADS,PRIMER_AMBIGUOUS_NUMREADS,PRIMER_COVERAGE,PRIMER_INPUT,NCOV_PCT_N_BASES,NCOV_PCT_COVERED_BASES,NCOV_LONGEST_NO_N_RUN,NCOV_NUM_ALIGNED_READS,NCOV_QC_PASS,PANGOLIN_LINEAGE,PANGOLIN_CONFLICT,PANGOLIN_AMBIGUITY_SCORE,PANGOLIN_SCORPIO_CALL,PANGOLIN_SCORPIO_SUPPORT,PANGOLIN_SCORPIO_CONFLICT,PANGOLIN_SCORPIO_NOTES,PANGOLIN_PANGO_DESIGNATION_VERSION,PANGOLIN_VERSION,PANGOLIN_SCORPIO_VERSION,PANGOLIN_CONSTELLATION_VERSION,PANGOLIN_IS_DESIGNATED,PANGOLIN_QC_STATUS,PANGOLIN_QC_NOTES,PANGOLIN_NOTE,PANGOLIN_DATA_VERSION
+0c0b5622-d63a-42cc-a804-13d71e1a5306,Completed,53046,ARTIC_V4-1,550,49,0,0.4757281553398058,unknown,1.11,98.89,19359,501636,True,B.1.369,0.0,,,,,,PUSHER-v1.26,4.3,0.3.17,v0.1.12,False,pass,Ambiguous_content:0.03,Usher placements: B.1.369(1/1),1.21
+39330daf-8dc6-4faa-a2d0-a2c1c909ab3b,Completed,63706,ARTIC_V4-1,471,47,0,0.4563106796116505,unknown,1.88,98.12,19372,494576,True,B.1,0.0,,,,,,PUSHER-v1.26,4.3,0.3.17,v0.1.12,False,pass,Ambiguous_content:0.03,Usher placements: B.1(2/2),1.21
+5695c67c-b7ac-4fc2-96f0-f1ffc68c03fb,Completed,9038,Midnight-ONT_V3,8439,29,0,1.0,unknown,0.79,99.17,22147,459268,True,BA.1,0.0,,Omicron (BA.1-like),0.92,0.0,scorpio call: Alt alleles 53; Ref alleles 0; Amb alleles 2; Oth alleles 3,PUSHER-v1.26,4.3,0.3.17,v0.1.12,False,pass,Ambiguous_content:0.02,Usher placements: BA.1(1/1),1.21
+70fd6fd3-6eed-48a4-8e5d-d59ecb1dbf32,Completed,1990720,NEB-VarSkip_v2b,4640,1,0,0.0133333333333333,unknown,0.0,0.0,0,0,False,Unassigned,,,,,,,PUSHER-v1.26,4.3,0.3.17,v0.1.12,False,fail,failed to map,,1.21
```

Comment:
it should be 1.21 (as we use ref data 1.26 in the CI)

Reply:
Changed to 1.26